test/test_taskdef_with_role.py

```diff
@@ -1,4 +1,7 @@
 import unittest
```

Comment:
This bit could do with a bit of [isorting](https://github.com/timothycrosley/isort)